spec/acceptance/nodesets/ubuntu-1404-docker.yml

```diff
@@ -0,0 +1,17 @@
+HOSTS:
+  ubuntu-1404:
+    roles:
+      - agent
+    platform: ubuntu-14.04-amd64
+    hypervisor : docker
+    image: ubuntu:14.04
+    docker_preserve_image: true
+    docker_cmd: '["/sbin/init"]'
+    docker_image_commands:
+      - 'rm /usr/sbin/policy-rc.d'
+      - 'rm /sbin/initctl; dpkg-divert --rename --remove /sbin/initctl'
+      - 'apt-get install -y -q net-tools wget'
+      - 'locale-gen en_US.UTF-8'
+CONFIG:
+  type: aio
```

Comment:
Why are some types `aio` and some are `foss`? What is the difference?

Reply:
Should probably be `foss`.  I pulled the ubuntu files from one of my other puppet modules and the files were actually a contribution.  I know the determination of what is installed is actually not in beaker and not from nodeset file, not sure if `type` even used but it is here: https://github.com/puppetlabs/beaker/blob/master/acceptance/fixtures/module/spec/acceptance/nodesets/default.yml

Real place Puppet install type is defined: https://github.com/puppetlabs/beaker-puppet_install_helper#run_puppet_install_helper

Probably just need to make them all `foss`.

Reply:
Please make them all `foss` if that's what they should be.